From cb6d0f6208283b93f1c49194c5e18ab5705b4b23 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Wed, 6 Feb 2019 13:22:34 +0000
Subject: [PATCH 0947/1077] MANTIS 0024715 Build: VC10; fix compiler warnings

warning C4996: 'access': The POSIX name for this item is deprecated. Instead, use the ISO C++ conformant name: _access. See online help for details.
warning C4996: 'getcwd': The POSIX name for this item is deprecated. Instead, use the ISO C++ conformant name: _getcwd. See online help for details.
warning C4996: 'strlwr': The POSIX name for this item is deprecated. Instead, use the ISO C++ conformant name: _strlwr. See online help for details.
warning C4996: 'strupr': The POSIX name for this item is deprecated. Instead, use the ISO C++ conformant name: _strupr. See online help for details.


git-svn-id: svn://localhost/SelfServiceCommon/trunk@1435 90b65887-3827-0410-9a23-83215b262276
---
 .../Massai/cpp/MassaiDLL/src/mConfig.cpp      | 24 +++++++++----------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/SelfServiceCommon/Massai/cpp/MassaiDLL/src/mConfig.cpp b/SelfServiceCommon/Massai/cpp/MassaiDLL/src/mConfig.cpp
index fc5e4dc4..c2bbbb13 100644
--- a/SelfServiceCommon/Massai/cpp/MassaiDLL/src/mConfig.cpp
+++ b/SelfServiceCommon/Massai/cpp/MassaiDLL/src/mConfig.cpp
@@ -908,14 +908,14 @@ CHAR   *pStr;
 
         // simply scan path environment var
 
-        pStr = getcwd(szPath,sizeof(szPath));
+        pStr = _getcwd(szPath, sizeof(szPath));
         debugWrite(HERE,"searching current working directory: %s\n",pStr);
 
         if (pStr)
         {
-          strcpy(szTmp,strlwr(pStr));
-          strcat(szTmp,"\\massai.cfg");
-          if(!access(szTmp,0))
+          strcpy(szTmp, _strlwr(pStr));
+          strcat(szTmp, "\\massai.cfg");
+          if(!_access(szTmp, 0))
           {
               fFound = TRUE;
               strcpy(szRootPath,szPath);
@@ -933,14 +933,14 @@ CHAR   *pStr;
           {
 
             debugWrite(HERE,"search config file in: %s\n",pStr);
-            strcpy(szTmp,strlwr(pStr));
+            strcpy(szTmp, _strlwr(pStr));
 
             for(fFound = FALSE, pToken = strtok(szTmp,";"); !fFound && pToken;)
             {
               strcpy(szPath,pToken);
               strcat(szPath,"\\massai.dll");
 
-              if(!access(szPath,0))
+              if(!_access(szPath, 0))
               {
                 fFound = TRUE;
 
@@ -952,13 +952,13 @@ CHAR   *pStr;
 
             if(!fFound)
             {
-              strcpy(szTmp,strlwr(pStr));
+              strcpy(szTmp, _strlwr(pStr));
               for(fFound = FALSE, pToken = strtok(szTmp,";"); !fFound && pToken;)
               {
                 strcpy(szPath,pToken);
                 strcat(szPath,"\\massai.cfg");
 
-                if(!access(szPath,0))
+                if(!_access(szPath, 0))
                 {
                   fFound = TRUE;
 
@@ -996,7 +996,7 @@ CHAR   *pStr;
 
       std::string testpath(szRootPath);
       testpath.append(MUTILS_CFG_FILE);
-      
+
       std::string userDir;
       if(!::PathFileExists(testpath.c_str()))
       {
@@ -1019,14 +1019,14 @@ CHAR   *pStr;
         {
           userDir.assign("");
         }
-       
+
       }
       if(userDir.size())
       {
         strncpy(szRootPath,userDir.c_str(),CHARBUF-1);
         szRootPath[CHARBUF-1]= '\0';
       }
-      
+
       strcpy(pszPath,szRootPath);
       return(0);
     }
@@ -1420,7 +1420,7 @@ BOOL rc = FALSE;
     {
       /* check uppercase string in Buffer for boolean value */
 
-      strupr(szBuffer);
+      _strupr(szBuffer);
 
       rc =    ((strcmp(szBuffer,"TRUE")    == 0) ||
                (strcmp(szBuffer,"YES")     == 0) ||
-- 
2.41.0.windows.1

